planner/core/flat_plan.go

```diff
@@ -401,6 +405,46 @@ func (f *FlatPhysicalPlan) flattenRecursively(p Plan, info *operatorCtx, target
 	return target, idx
 }
 
+func (f *FlatPhysicalPlan) flattenForeignKeyChecksAndCascadesMap(childCtx *operatorCtx, target FlatPlanTree, childIdxs []int, fkChecksMap map[int64][]*FKCheck, fkCascadesMap map[int64][]*FKCascade) (FlatPlanTree, []int) {
+	tids := make(sortutil.Int64Slice, 0, len(fkChecksMap))
+	for tid := range fkChecksMap {
+		tids = append(tids, tid)
+	}
+	// Sort by table id for explain result stable.
+	tids.Sort()
+	for _, tid := range tids {
+		target, childIdxs = f.flattenForeignKeyChecksAndCascades(childCtx, target, childIdxs, fkChecksMap[tid], nil)
+	}
+	tids = tids[:0]
+	for tid := range fkCascadesMap {
+		tids = append(tids, tid)
+	}
+	tids.Sort()
```

Comment:
Now we can use `slices.Sort` from `golang.org/x/exp/slices` to sort a slice of numbers.

Reply:
done.